generators/theme/generate.php

```diff
@@ -1,8 +1,12 @@
 <?php
 
+use Dxw\Whippet\Git\Git;
+
 class ThemeGenerator extends \Dxw\Whippet\WhippetGenerator {
   use \Dxw\Whippet\Modules\Helpers\WhippetHelpers;
 
+  protected $whippet_theme_repo = 'git@github.com:dxw/whippet-theme-template.git';
```

Comment:
Could we use the https:// clone URL instead? That way non-dxw employees can use this too (this is a public repo, after all).